pkg/k8s/api.go

```diff
@@ -192,7 +192,7 @@ func GetPodStatus(pod corev1.Pod) string {
 	for i := range pod.Status.InitContainerStatuses {
 		container := pod.Status.InitContainerStatuses[i]
 		switch {
-		case container.State.Terminated != nil && container.State.Terminated.ExitCode == 0 && container.State.Terminated.Signal == 0:
+		case container.State.Terminated != nil && container.State.Terminated.ExitCode == 0:
```

Comment:
@alpeb Already commented on this above but do we want to ignore all signals now? I get ignoring `9`, but maybe others may be more significant? Not a blocker, but just curious if this is definitely what we want now.